setup.py

```diff
@@ -120,6 +120,8 @@ def run(self):
   libs.extend(['libngraph.so', 'libcodegen.so', 'libcpu_backend.so', 'libmkldnn.so', 'libtbb_debug.so', 'libtbb_debug.so.2', 'libtbb.so', 'libtbb.so.2'])
   # Nuphar Libs
   libs.extend(['libtvm.so.0.5.1'])
+  # Openvino Libs
+  libs.extend(['libcpu_extension.so'])
```

Comment:
we don't want to add this openvino lib to the python package right? besides, there are more dependencies than that single library.